mods/ts/languages/rules/en.ftl

```diff
@@ -286,199 +286,199 @@ meta-tree-name = Tree
 meta-rock-name = Rock
 meta-box-name = Box
 meta-drum-name = Drum
-meta-palette-name = Palette
+meta-palette-name = Pallet
 meta-railway-name = Railway
-meta-gate-description = Automated barrier that opens for allied units.
+meta-gate-description = Automated gate that opens for allied units
```

Comment:
You are explaining by re-using the term itself.

Reply:
A gate doesn’t need much explanation,  more precise as not all barriers are gates.